Frameworks/UIKit/StarboardXaml/ApplicationCompositor.cpp

```diff
@@ -39,14 +39,8 @@ void InitializeApp() {
     initialized = true;
 
     // Set our writable and temp folders
-    auto pathData = Windows::Storage::ApplicationData::Current->LocalFolder->Path;
-    IwSetWritableFolder(pathData->Data());
-
-    char writableFolder[2048];
-    size_t outLen;
-    auto tempPathData = Windows::Storage::ApplicationData::Current->TemporaryFolder->Path;
-    wcstombs_s(&outLen, writableFolder, tempPathData->Data(), sizeof(writableFolder) - 1);
-    SetTemporaryFolder(writableFolder);
+    IwSetWritableFolder(Windows::Storage::ApplicationData::Current->LocalFolder->Path->Data());
+    SetTemporaryFolder(Windows::Storage::ApplicationData::Current->TemporaryFolder->Path->Data());
```

Comment:
They are both silly. It doesn't make sense that the application compositor sets any sort of file system stuff. Starboard should know how to do this itself. That would be more proper encapsulation ... 

---
In reply to: [99649338](https://github.com/Microsoft/WinObjC/pull/1916#discussion_r99649338) [](ancestors = 99649338)